wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -557,8 +557,54 @@ function search_oxford_dictionary(){
 /** 
  * Creating shortcode for students
  * */
-function students_shortcode() {
+function students_shortcode( $attributes ) {
```

Comment:
You actually don't what your shortcode to output any HTML. You need to return that HTML instead.

You should look into `ob_start()` and `ob_get_clean()`.
https://www.php.net/manual/en/function.ob-start.php
https://www.php.net/manual/en/function.ob-get-clean.php

Reply:
Directly outputting the HTML can lead to some really unexpected behaviour.

Reply:
your favourite `ob_start()` :D :D :D